cli/tests/unit/write_text_file_test.ts

```diff
@@ -1,4 +1,12 @@
 import { unitTest, assert, assertEquals } from "./test_util.ts";
+import { resolve, join } from "../../../std/path/mod.ts";
+
+const getResolvedUrl = (path: string): URL =>
+  new URL(
+    Deno.build.os === "windows"
+      ? "file:///" + resolve(path).replace(/\\/g, "/")
+      : "file://" + resolve(path)
```

Comment:
This bit of code which changes the backslashes to forward slashes is repeated a lot throughout the tests. It would be good to have a utility function for this.

Reply:
I've moved it to cli/tests/unit/test_util.ts, if there's a better place to put it please let me know

Reply:
Judging by a few of these PR comments, it seems like the most important piece of this function was being masked by the way it's written. The important bit is the three slashes for windows and two slashes for *nix (actually it turns out the backslash replacement isn't even necessary).

I've removed this piece of logic entirely now, and I think all the tests are more readable for it, please let me know what you think.